changelogs/fragments/6763-keycloak-auth-provider-choices.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Added provider ID choices, since Keycloak supports only those two specific ones (https://github.com/ansible-collections/community.general/pull/6763).
```

Comment:
```suggestion
  - keycloak_authentication - added provider ID choices, since Keycloak supports only those two specific ones (https://github.com/ansible-collections/community.general/pull/6763).
```